examples/hello_fletcher/hw/hello_fletcher.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": 1,
+  "afu-image": {
+    "power": 0,
+    "afu-top-interface": {
+      "name": "ofs_plat_afu"
```

Comment:
This is the important line: this is the NEW (v2) PIM (platform interface manager)